server/play/model/reducers/NewGameStateReducer.ts

```diff
@@ -52,10 +52,10 @@ const handleMarkPlayerReadyAction = (state: NewGameBoardState, action: PlayerRea
     const { dog, hands } = dealCards(state.players.length);
     const playerOrder = shufflePlayers(state.players);
 
-    // For debugging shows - give player Greg Cole a hand that can show.
-    // const { dog, hands } = SampleDeal;
-    // const playerOrder = shuffle(without(state.players, "Greg Cole"));
-    // playerOrder.push("Greg Cole");
+    // Give greg cole a hand with four kings
```

Comment:
Can we move this to a test instead of manually uncommenting this?

Reply:
We could also write a test, but this is useful for end to end level testing. This is how you would testing something in a real game if you needed to, which is why I'd like to also leave this here.

Reply:
OK we will FLUP with https://github.com/gillessed/palantarot/issues/92